apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/error/page.tsx

```diff
@@ -30,48 +32,6 @@ export const metadata: Metadata = {
     'There was an error processing your subscription. If this problem persists, please contact support.',
 };
 
-const getErrorReason = (
```

Comment:
Since this is used in multiple locations, I believe it's worth pulling into a separate function.